README.md

```diff
@@ -20,10 +20,10 @@ The LUKSO CLI is able to install multiple clients for running the node.
 
 | Client     | Version | Release                                                      |
 |------------|---------|--------------------------------------------------------------|
-| Geth       | v1.12.2 | https://github.com/ethereum/go-ethereum/releases/tag/v1.12.2 |
-| Erigon     | v2.48.1 | https://github.com/ledgerwatch/erigon/releases/tag/v2.48.1   |
+| Geth       | v1.13.1 | https://github.com/ethereum/go-ethereum/releases/tag/v1.13.1 |
+| Erigon     | v2.49.2 | https://github.com/ledgerwatch/erigon/releases/tag/v2.49.2   |
 | Prysm      | v4.0.8  | https://github.com/prysmaticlabs/prysm/releases/tag/v4.0.8   |
-| Lighthouse | v4.3.0  | https://github.com/sigp/lighthouse/releases/tag/v4.3.0       |
+| Lighthouse | v4.4.1  | https://github.com/sigp/lighthouse/releases/tag/v4.4.1       |
 | Teku       | v23.8.0 | https://github.com/Consensys/teku/releases/tag/23.8.0        |
```

Comment:
I forgot to mention `Teku`. I think that we should update to `23.9.0`: https://github.com/Consensys/teku/releases/tag/23.9.0

Reply:
Added and tested - all good there :+1: